example-model/src/main/java/uk/gov/gchq/palisade/example/config/ApplicationConfiguration.java

```diff
@@ -28,12 +28,12 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.context.annotation.Primary;
 
+import uk.gov.gchq.palisade.clients.simpleclient.web.DataClient;
+import uk.gov.gchq.palisade.clients.simpleclient.web.PalisadeClient;
```

Comment:
I would argue that these should be example copies of those client interfaces which would decouple the clients repo from the configuration of the example. however that will get solved by a different ticket soon. So happy to leave as is for now.

Reply:
For feign reasons, I'm not sure how easy this is